package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "name-request",
-  "version": "2.0.26",
+  "version": "2.1.01",
```

Comment:
I don't think the leading zero is a problem but we've never done that before.

Reply:
haha I was wondering about that - I'll change it